build.gradle.kts

```diff
@@ -1,17 +1,15 @@
 plugins {
     java
     `java-library`
-    signing
-    `maven-publish`
-    checkstyle
-    id("org.gradle.crypto.checksum") version "1.4.0"
-    id("io.github.gradle-nexus.publish-plugin") version "1.1.0"
-    jacoco
 }
 
 val projectGroup: String by project
 val swagger: String by project
 val rsApi: String by project
+val edcGroup: String by project
+val annotationProcessorVersion: String by project
+val javaVersion: String by project
+val metaModelVersion: String by project
 
 // these values are required for the project POM (for publishing)
 val edcDeveloperId: String by project
```

Comment:
developer id, name, and email are not needed, can be deleted, also from `gradle.properties`